AIDevGallery/Samples/ModelsDefinitions/embeddings.modelgroup.json

```diff
@@ -18,7 +18,8 @@
             "Description": "This is an embedding model executed on the CPU or GPU",
             "HardwareAccelerator": [
               "CPU",
-              "DML"
+              "DML",
+              "QNN"
```

Comment:
we need to do some testing here before enabling the models for qnn - I'm seeing both the cpu and npu running which might mean the model is falling back to cpu and it's not optimized for qnn

Reply:
removed qnn from embeddings models